stable/sentry/values.yaml

```diff
@@ -108,9 +110,13 @@ ingress:
 # TODO: add support for plugins https://docs.sentry.io/server/plugins/
 
 postgresql:
+  enabled: false
```

Comment:
imho this will be confusing for the user. Couldn't you just check that `postgresHost` is set?
You should update `README.md` with these too.

Reply:
I see this pattern in other charts, I think it make sense to "disable internal PG to use external one" as you don't want both at same time

Reply:
Indeed, this is what other charts seem to do. I still don't think that's the best solution (because at first sight the user might think that's this is a variable to disable the postgresql database all together), but it still needs to be documented in the README.md :)